src/generated/CMakeLists.txt

```diff
@@ -10,10 +10,10 @@ if(QUIC_ENABLE_LOGGING)
         add_library(logging INTERFACE)
     else()
         target_include_directories(logging_inc INTERFACE ${CMAKE_CURRENT_SOURCE_DIR}/common)
-        target_include_directories(logging_inc INTERFACE ${CMAKE_CURRENT_SOURCE_DIR}/${QUIC_LTTNG_PLATFORM})
+        target_include_directories(logging_inc INTERFACE ${CMAKE_CURRENT_SOURCE_DIR}/${QUIC_LINUX_LOGGING_METHOD})
```

Comment:
Maybe we default `QUIC_LINUX_LOGGING_METHOD` to lttng and then the build.ps1 doesn't even need to set it?